content/docs/0.8.x/integrations/migrate_from_070_to_080/index.md

```diff
@@ -0,0 +1,739 @@
+    const ServiceName = "keptn-service-template-go" // replace with your service name
+    ```
+* **Introduce cloud-event parse function** with error handling, to reduce code duplication
+    ```golang
+    /**
+     * Parses a Keptn Cloud Event payload (data attribute)
+     */
+    func parseKeptnCloudEventPayload(event cloudevents.Event, data interface{}) error {
+        err := event.DataAs(data)
+        if err != nil {
+            log.Fatalf("Got Data Error: %s", err.Error())
+            return err
+        }
+        return nil
+    }
+    ```
+* **Use keptnv2** for most things, e.g.:
+    ```golang
+    import (
+        keptnv2 "github.com/keptn/go-utils/pkg/lib/v0_2_0"
+    )
+    
+    myKeptn, err := keptnv2.NewKeptn(&event, keptnOptions)
+    
+    switch event.Type() {
+    
+        // -------------------------------------------------------
+        case keptnv2.GetTriggeredEventType(keptnv2.TestTaskName): // sh.keptn.event.test.triggered
+        log.Printf("Processing Test.Triggered Event")
+        
+        eventData := &keptnv2.TestTriggeredEventData{}
+        parseKeptnCloudEventPayload(event, eventData)
+        
+        return HandleTestsTriggered(myKeptn, event, eventData)
+        ...
+    }
+    ```
+* **func _main(...) uses cloudevents.NewClient** to run the http events endpoint
+    ```golang
+    func _main(args []string, env envConfig) int {
+        // configure keptn options
+        if env.Env == "local" {
+            log.Println("env=local: Running with local filesystem to fetch resources")
+            keptnOptions.UseLocalFileSystem = true
+        }
+    
+        keptnOptions.ConfigurationServiceURL = env.ConfigurationServiceUrl
+        
+        ctx := context.Background()
+        ctx = cloudevents.WithEncodingStructured(ctx)
+        
+        // configure http server to receive cloudevents
+        p, err := cloudevents.NewHTTP(cloudevents.WithPath(env.Path), cloudevents.WithPort(env.Port))
+    
+        if err != nil {
+            log.Fatalf("failed to create client, %v", err)
+        }
+        c, err := cloudevents.NewClient(p)
+        if err != nil {
+            log.Fatalf("failed to create client, %v", err)
+        }
+    
+        log.Printf("Starting receiver")
+        log.Fatal(c.StartReceiver(ctx, processKeptnCloudEvent))
+    
+        return 0
+    }
+    ```
+
+
+### eventhandlers.go
+
+We have introduced a new helper function to log CloudEvents to console:
+```golang
+// GenericLogKeptnCloudEventHandler is a generic handler for Keptn Cloud Events that logs the CloudEvent
+func GenericLogKeptnCloudEventHandler(myKeptn *keptnv2.Keptn, incomingEvent cloudevents.Event, data interface{}) error {
+	log.Printf("Handling %s Event: %s", incomingEvent.Type(), incomingEvent.Context.GetID())
+	log.Printf("CloudEvent %T: %v", data, data)
+
+	return nil
+}
+```
+Other than that, please follow the changes detailed above for Keptn CloudEvents, `go-utils` and `distributor`, as shown above.
```

Comment:
Other than that, please follow the changes detailed above for Keptn CloudEvents, `go-utils` and `distributor`, ~~as shown above~~.